documentation/specs/event-source.md

```diff
@@ -34,6 +34,8 @@ EventSource is primarily used to profile code. For MSBuild specifically, a major
 | Save | Saves a project to the file system if dirty, creating directories as necessary. |
 | SdkResolverResolveSdk | A single SDK resolver is called. |
 | SdkResolverServiceInitialize | Initializes SDK resolvers. |
+| CachedSdkResolverServiceResolveSdk | The caching SDK resolver service is resolving an SDK. |
```

Comment:
nit:
Alphabetized
It's reasonable to want thinks that are related to each other to be together, in which case I'd be open to renaming this "SdkResolverServiceResolveSdkFromCache"

Reply:
Actually, the name is already in this list. I must have messed it up when I did the revert+changes, so I'm going to undo this.